spread.yaml

```diff
@@ -28,6 +28,8 @@ environment:
     SUDO_UID: ""
     TRUST_TEST_KEYS: '$(HOST: echo "${SPREAD_TRUST_TEST_KEYS:-true}")'
     MANAGED_DEVICE: "false"
+    # TODO: change back to latest/stable when 4.0 is fixed
```

Comment:
:heart:

Reply:
and a PR to switch back when we can: https://github.com/snapcore/snapd/pull/8505